app/initializers/ember-cli-mirage.js

```diff
@@ -23,7 +25,7 @@ export function startMirage(env = ENV) {
   let modules = readModules(env.modulePrefix);
   let options = _assign(modules, {environment, baseConfig, testConfig});
 
-  return new Server(options);
+  return new Server(Object.assign({createInterceptor: createPretender}, options));
```

Comment:
Let's use Lodash assign, since Object.assign doesn't work in Phantomjs
```
import _assign from 'lodash/object/assign';

_assign({ createInterceptor: createPretender }, options));
```